docs/resources/aviatrix_controller_access_allow_list_config.md

```diff
@@ -0,0 +1,48 @@
+---
+subcategory: "Settings"
+layout: "aviatrix"
+page_title: "Aviatrix: aviatrix_controller_access_allow_list_config"
+description: |-
+  Creates and manages an Aviatrix Controller's Access Allow List Config
+---
+
+# aviatrix_controller_access_allow_list_config
+
+!> **WARNING:** If any of the Allow List IPs are incorrect, the Controller will be inaccessible.
+
+The **aviatrix_controller_access_allow_list_config** resource allows management of an Aviatrix Controller's Access Allow List.
+
+## Example Usage
+
+```hcl
+# Create an Aviatrix Controller Access Allow List Config
+resource "aviatrix_controller_access_allow_list_config" "test" {
+  allow_list {
+    ip_address = "0.0.0.0"
+  }
```

Comment:
Example should probably not show 0.0.0.0, otherwise, there would be no need for the 2nd rule in the example. We can also add `description` within the example as well